### DISCUSSION TITLE
Startup state of WezTerm on macOS

### DISCUSSION THREAD

**Discussion by carlca** [wezterm/wezterm#5916] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

Forgive me if I have missed something obvious, but is there a way of specifying the startup window position and size when the app starts? I'm forever having to resize WezTerm to a bigger window, and I'm sure there must be some way to do this!

I've searched the issues on Github and consequently tried adding the `initial_cols` and `initial_rows` settings to `.wezterm.lua` but it's not making any difference! What am I missing?

This is what is coming up in `.wezterm.lua`...

![SCR-20240808-muay](https://github.com/user-attachments/assets/4c43b2df-ccfd-4a9e-9f42-e5d5c1249fed)

**Comment by wez** [comment, marked as answer]
You need to assign the configuration in the config object:

`config.initial_rows  = 60`

**Comment by carlca** [comment]
Thanks Wez! I'm embarrassed to say that I did, indeed, miss something obvious. Unforgivable considering the other lines all had the `config.` construct :blush:
